src/main/java/io/jenkins/plugins/orka/AgentTemplate.java

```diff
@@ -102,25 +103,25 @@ public AgentTemplate(String vmCredentialsId, String vm, boolean createNewVMConfi
             String baseImage, int numCPUs, int numExecutors, String remoteFS, Mode mode, String labelString,
             String namePrefix, RetentionStrategy<?> retentionStrategy, OrkaVerificationStrategy verificationStrategy,
             List<? extends NodeProperty<?>> nodeProperties, String jvmOptions, String scheduler, String memory) {
-        this(vmCredentialsId, vm, createNewVMConfig, configName, baseImage, numCPUs, false, numExecutors, remoteFS,
-                mode, labelString, namePrefix, retentionStrategy, verificationStrategy, nodeProperties, jvmOptions,
-                scheduler, memory);
+        this(vmCredentialsId, vm, createNewVMConfig, configName, baseImage, numCPUs, false, false, 
+                numExecutors, remoteFS, mode, labelString, namePrefix, retentionStrategy, verificationStrategy, 
+                nodeProperties, jvmOptions, scheduler, memory);
     }
 
     public AgentTemplate(String vmCredentialsId, String vm, boolean createNewVMConfig, String configName,
-            String baseImage, int numCPUs, boolean useNetBoost, int numExecutors, String remoteFS, Mode mode,
-            String labelString, String namePrefix, RetentionStrategy<?> retentionStrategy,
+            String baseImage, int numCPUs, boolean useNetBoost, boolean useGpuPassthrough, int numExecutors, 
+            String remoteFS, Mode mode, String labelString, String namePrefix, RetentionStrategy<?> retentionStrategy,
             OrkaVerificationStrategy verificationStrategy, List<? extends NodeProperty<?>> nodeProperties,
             String jvmOptions, String scheduler, String memory) {
-        this(vmCredentialsId, vm, createNewVMConfig, configName, baseImage, numCPUs, useNetBoost, numExecutors,
-                remoteFS, mode, labelString, namePrefix, retentionStrategy, verificationStrategy, nodeProperties,
-                jvmOptions, scheduler, memory, false, null, null);
+        this(vmCredentialsId, vm, createNewVMConfig, configName, baseImage, numCPUs, useNetBoost, useGpuPassthrough, 
+                numExecutors, remoteFS, mode, labelString, namePrefix, retentionStrategy, verificationStrategy, 
+                nodeProperties, jvmOptions, scheduler, memory, false, null, null);
     }
 
     @DataBoundConstructor
     public AgentTemplate(String vmCredentialsId, String vm, boolean createNewVMConfig, String configName,
-            String baseImage, int numCPUs, boolean useNetBoost, int numExecutors, String remoteFS, Mode mode,
-            String labelString, String namePrefix, RetentionStrategy<?> retentionStrategy,
+            String baseImage, int numCPUs, boolean useNetBoost, boolean useGpuPassthrough, int numExecutors, 
+            String remoteFS, Mode mode, String labelString, String namePrefix, RetentionStrategy<?> retentionStrategy,
```

Comment:
This is a breaking change.
A lot of users are configuring the plugin directly with code. And if we are to do it this way, we would break their automation.
We need to introduce a new overload and keep the old one